### ISSUE TITLE
ServiceParser regexp extract params on same line

### ISSUE THREAD

**Issue by biesbjerg** [biesbjerg/ng2-translate-extract#14]
ServiceParser regexp extract params on same line.

**Run extract**
```
this._translateService.get('You are expected at {{time}}', {time: moment.format('H:mm')}).subscribe();
```

**Expected:**
`You are expected at {{time}}`

**Actual result:**
`You are expected at {{time}}', {time: moment.format('H:mm`

Any regexp experts want to help with this?
https://regex101.com/r/NeEU3q/1

**Comment by P-H-F** [comment]
I'm not sure if that helps, but you could still split it up in two separate Regex' based on how many parameters are present.
(I tried to find a solution where both are merged, but I was not able to do it)

The one below works if both parameters are available.

https://regex101.com/r/ulcJE6/6

**Comment by biesbjerg** [comment]
I'm experimenting with parsing typescript to an AST using the compiler API. Should be a lot more robust than regular expressions, when I hopefully get the details worked out. Stay tuned!

**Comment by biesbjerg** [comment]
I just released version 0.6.0 with an experimental AstServiceParser, that hopefully is more robust (I need help testing it).

Use it like this:
`ng2-translate-extract --experimental`
